R/configureApp.R

```diff
@@ -62,13 +62,7 @@ configureApp <- function(
     propertyValue <- properties[[i]]
 
     # dispatch to the appropriate client implementation
-    if (is.function(client$configureApplication)) {
```

Comment:
This is a shinyapps function, but the only `configureApplication()` method was in the Connect client, and it didn't take these arguments (and didn't actually configure anything). Deleted.